features/mcp/00-mcp-worker-rt.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-rt
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,worker-rt]}
```

Comment:
hm I can see you want to apply machine configs for worker and worker-rt labels, but I thought the worker machine-config-pool will apply worker machine configs, so why do we need to apply it twice?

Reply:
i believe it's required by mcp

Reply:
probably because of the bug https://bugzilla.redhat.com/show_bug.cgi?id=1772490